CHANGELOG.md

```diff
@@ -18,18 +18,21 @@ NOTE: We are deprecating several collectors in this release.
 * [FEATURE] Add ZFS collector + review feedback from PRs 213 and 369 #410
```

Comment:
Agreed that it doesn't add value in the changelog. Sorry for injecting it, the problem shouldn't reoccur.